<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addProd</class>
 <widget class="QDialog" name="addProd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Производитель:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="brendCmb"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Тип:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typrCmb">
         <item>
          <property name="text">
           <string>Легковой</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Грузовой</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Мотоцикл</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Водный</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Модель:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="modelEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Объем двигателя:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="VSb"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Топливо:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fuelCmb"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Коробка передач:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="transCmb"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Мощность:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="powerSb"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Год производства:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yearSb"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Привод:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="unitCmb"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Цвет:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="colorEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Цвет салона:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inColorEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Материал салона:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="matEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Цена:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="priceBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Количество:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="amountLb">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addProd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addProd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
